common/src/commonMain/kotlin/io/xorum/codeforceswatcher/util/IFeedbackController.kt

```diff
@@ -0,0 +1,78 @@
+package io.xorum.codeforceswatcher.util
+
+abstract class IFeedbackController {
+    abstract var countScreenOpening: Int
+
+    abstract var startTimeWhenShown: Long
+
+    abstract var isNeededAtAll: Boolean
+
+    abstract var isLockoutPeriod: Boolean
+
+    abstract var currentShowingItem: Int
+
+    var feedbackItem: FeedbackItem
+        get() = when(currentShowingItem) {
+            0 -> buildEnjoyingItem()
+            1 -> { isNeededAtAll = false; buildEmailItem() }
```

Comment:
`buildEmailItem` here?

Reply:
not sure i understand

Reply:
Ah, I've missed the second call, because it's followed after `;`. The more readable version would be `buildEmailItem ().also { isNeededAtAll = false }`.